tests/unit_tests/test_wave_06.py

```diff
@@ -33,12 +33,11 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("Electronics")
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert len(items) == 0
```

Comment:
For this test, the first assertion statement is all you need to confirm that items is an empty list.